config/babel.prod.js

```diff
@@ -10,20 +10,20 @@
 module.exports = {
   babelrc: false,
   presets: [
-    'babel-preset-es2015',
-    'babel-preset-es2016',
-    'babel-preset-react'
-  ].map(require.resolve),
+    require.resolve('babel-preset-es2015'),
+    require.resolve('babel-preset-es2016'),
+    require.resolve('babel-preset-react')
+  ],
   plugins: [
-    'babel-plugin-syntax-trailing-function-commas',
-    'babel-plugin-transform-class-properties',
-    'babel-plugin-transform-object-rest-spread',
-    'babel-plugin-transform-react-constant-elements',
-  ].map(require.resolve).concat([
+    require.resolve('babel-plugin-syntax-trailing-function-commas'),
+    require.resolve('babel-plugin-transform-async-to-generator'),
```

Comment:
Yeah, PR would be welcome. I had no idea what’s the right way to integrate this. 😄